src/ngMock/angular-mocks.js

```diff
@@ -984,7 +984,7 @@ angular.mock.dump = function(object) {
     } else if (angular.isObject(object)) {
       if (angular.isFunction(object.$eval) && angular.isFunction(object.$apply)) {
         out = serializeScope(object);
-      } else if (object instanceof Error) {
+      } else if (angular.isError(object)) {
```

Comment:
We could make it "private API" (e.g. `angular.$$isError`), but I don't think it is worth it. This function (`angular.mock.dump`) is very rarely used. It is also very rare to have new window contexts in unit tests and it is even more rare to have them throw errors. Plus, `DOMException`s get decently stringified by `JSON.stringify` (in contrast to plain `Error`s which become `"{}"`). All that considered, I think this one occurrence can remain `object instanceof Error` :smiley: